marlowe-playground-client/src/MetadataTab/View.purs

```diff
@@ -41,76 +41,82 @@ onlyDescriptionRenderer setAction deleteAction key info needed metadataAction ty
   ]
     <> if needed then [] else [ div [ classes [ ClassName "metadata-error", ClassName "metadata-prop-not-used" ] ] [ text "Not used" ] ]
 
-choiceMetadataRenderer :: forall a p. String -> ChoiceInfo -> Boolean -> (MetadataAction -> a) -> String -> String -> Array (HTML p a)
-choiceMetadataRenderer key info@{ choiceFormat } needed metadataAction typeNameTitle typeNameSmall =
+type FormattedNumberActions
+  = { setFormat :: String -> NumberFormat -> MetadataAction
+    , setDescription :: String -> String -> MetadataAction
+    , deleteInfo :: String -> MetadataAction
+    }
+
+formattedNumberMetadataRenderer :: forall a p. String -> String -> NumberFormat -> FormattedNumberActions -> Boolean -> (MetadataAction -> a) -> String -> String -> Array (HTML p a)
```

Comment:
we have quite the amount of arguments here and 4 of them are strings. If we are not relying on currying and partial application, I'd suggest to switch to an Object in a future PR and add some comments to the function